python/ray/serve/_private/http_state.py

```diff
@@ -121,6 +126,10 @@ def update(self):
             self._health_check_obj_ref = self._actor_handle.check_health.remote()
             self._last_health_check_time = time.time()
 
+    def shutdown(self):
+        self._deleting = True
```

Comment:
nit
```suggestion
        self._shutting_down = True
```
or `self._shut_down`

or `self._shutdown`

but better to match the method names